schema/deploy/shipping/views.sql

```diff
@@ -2911,4 +2913,200 @@ comment on view shipping.uw_reopening_ehs_reporting_v1 is
+            (select trim(string_response[1]) from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'testing_trigger') as testing_trigger,
+            (select trim(string_response[1]) from shipping.fhir_questionnaire_responses_v1 where encounter_id = all_uw_instances.encounter_id and link_id = 'surge_selected') as surge_selected,
+            sample.collected::date as sample_collection_date
+        from all_uw_instances
+        left join warehouse.sample using (encounter_id)
+        where redcap_event_name = 'encounter_arm_1'
+    ),
+
+    uw_individuals as (
+        select
+            individual,
+            max(uw_encounters.encountered) filter (where testing_trigger = '1') as latest_invite_date,
+            max(sample_collection_date) filter (where sample_collection_date is not null) as latest_collection_date
+        from uw_enrollments
+        left join uw_encounters using (individual)
+        group by individual
+    ),
+
+    -- Select encounters for testing based on positive daily attestations
+    positive_daily_attestations as (
+        select
+            uw_encounters.redcap_project_id,
+            uw_encounters.redcap_record_id,
+            uw_encounters.redcap_event_name,
+            uw_encounters.redcap_repeat_instance,
+            uw_encounters.encountered,
+            individual,
+            tier,
+            latest_invite_date,
+            latest_collection_date,
+            case
+                when daily_symptoms then 1
+                when daily_exposure_known_pos then 2
+                when daily_exposure then 3
+                else null
+            end as priority,
+            case
+                when daily_symptoms then 'symptomatic'
+                when daily_exposure_known_pos then 'exposure_to_known_positive'
+                when daily_exposure then 'gathering_over_10'
+                else null
+            end as priority_reason
+        from uw_encounters
+        join uw_individuals using (individual)
+        join uw_enrollments using (individual)
+        -- Filter to encounters for participants whose last invite was over 7 days before encounter
+        where (latest_invite_date is null or latest_invite_date < uw_encounters.encountered - interval '7 days')
+        -- Filter to encounters for participants who have never had a sample collected or their last sample collection was over 7 days before encounter
+        and (latest_collection_date is null or latest_collection_date < uw_encounters.encountered - interval '7 days')
+        -- Filter for instances that do no already have testing_trigger filled (null or empty string)
+        and (testing_trigger is null or testing_trigger = '')
+        -- Filter for postive daily attestations only
+        and screen_positive
```

Comment:
Ah, just remembering that we'll need to be more precise than this. For attestations where `daily_exposure_known_pos is true`, we only want to offer testing after a lag of X days. If we test too early after exposure, we risk negatives before there's enough virus to detect. I believe Trevor quoted 2-3 days as the optimum value for X, but we should confirm that value.

Reply:
Ah, right. Do we need lag for both `daily_exposure_known_pos` and `daily_exposure`?
I can incoporate this into the case statement:
```sql
case
    when daily_symptoms then 1
    when daily_exposure_known_pos and current_date - uw_encounters.encountered >= 3  then 2
    when daily_exposure and current_date - uw_encounters.encountered >= 3 then 3
    else null
 end as priority,
```

Reply:
Yes, both daily exposure questions. That case looks good to me, although I think interval conditions like this read easier:

```
age(uw_encounters.encountered) >= '3 days'
```

Reply:
I've [asked Trevor in #informatics](https://seattle-flu-study.slack.com/archives/CCAA9RBFS/p1602628191080200).

Reply:
Updated case statement to only include exposure for `age(uw_encounters.encountered) >= '2 days'`